### PR TITLE
replace master with main

### PR DESCRIPTION
**Pull request by motakbiri**
replacing word `master` with a more inclusive alternative (`main`) to remove unnecessary references to slavery

### CODE REVIEW

**Comment by kettanaito** [comment]
Hey, @motakbiri. Thanks for opening this. Sorry I haven't got to this earlier. We'd love to support this change, and I wanted to get this merged when we merge some other pending work to prevent fork confusion in Git. I haven't done this migration in an existing project, so I was not sure about the implications for other contributors. 

Did you find any obstacle that made you close this?

**Comment by kettanaito** [comment]
Okay, so I've assumed this referred to the branches in Git 🤦🏻 My bad.

File: src/mockServiceWorker.js [hunk 1]
```diff
@@ -83,11 +83,11 @@ self.addEventListener('message', async function (event) {
   }
 })
 
-// Resolve the "master" client for the given event.
+// Resolve the "main" client for the given event.
 // Client that issues a request doesn't necessarily equal the client
 // that registered the worker. It's with the latter the worker should
 // communicate with during the response resolving phase.
-async function resolveMasterClient(event) {
+async function resolveMainClient(event) {
```
Comment (kettanaito) [inline src/mockServiceWorker.js:90]: What do you think about `resolveTopClient`? That'd be much more accurate to what the function does.

**Comment by motakbiri** [comment]
Hi @kettanaito yep I realized I had created this PR from master branch. I’ll create another PR with the right branch and the naming you’ve suggested 👍

**Pull request closed without merging**